derive_builder_core/src/setter.rs

```diff
@@ -151,7 +154,26 @@ impl<'a> ToTokens for Setter<'a> {
                         let mut new = #self_into_return_ty;
                         new.#field_ident = ::derive_builder::export::core::option::Option::Some(converted);
                         Ok(new)
-                }));
+                    }
+                ));
+            }
+
+            if let Some(ref ident_each) = self.each {
+                tokens.append_all(quote!(
+                    #(#attrs)*
+                    #[allow(unused_mut)]
+                    #vis fn #ident_each <ITEM>(#self_param, item: ITEM) -> #return_ty
```

Comment:
Should this honor `setter(into)`, or is that going to cause type inference problems trying to figure out `ITEM`?

Reply:
Yeah, type inference doesn't work well here.

Additionally it requires to introduce one more type parameter `VALUE: Into<ACTUAL_VALUE>`. Type parameter may be replaced by an `impl Trait` approach, but, IIRC, this crate still targets 2015 edition, so this will not work either.